api/app/users/api.py

```diff
@@ -545,3 +554,31 @@ def get(self):
             UserComment.user_id == args['user_id']).all()
 
         return comments
+
+class UserApplicationCommentReviewAPI(restful.Resource):
+    @auth_required
+    @marshal_with(user_comment_review_fields)
+    def get(self):
+        req_parser = reqparse.RequestParser()
+        req_parser.add_argument('event_id', type=int, required=True)
+        req_parser.add_argument('user_id', type=int, required=True)
+        args = req_parser.parse_args()
+
+        current_user = user_repository.get_by_id(g.current_user['id'])
+        if not current_user.is_event_admin(args['event_id']):
+            return FORBIDDEN
+
+        comments = db.session.query(UserComment).filter(
+            UserComment.event_id == args['event_id'],
+            UserComment.user_id == args['user_id']).all()
+        
+        review_by_user_firstname = user_repository.get_by_id(args['user_id']).firstname
```

Comment:
So, who is this getting? The current users's firstname? 
Should we not be getting the first name of the reviewers who made the comments?